vertigo-bot-extensions/vertigo-bot-confluence/src/main/java/io/vertigo/chatbot/engine/plugins/bt/confluence/ConfluencePlugin.java

```diff
@@ -0,0 +1,22 @@
+package io.vertigo.chatbot.engine.plugins.bt.confluence;
+
+import javax.inject.Inject;
+
+import io.vertigo.chatbot.engine.plugins.bt.confluence.multilingual.ConfluenceMultilingualResources;
+import io.vertigo.chatbot.executor.ExecutorPlugin;
+import io.vertigo.core.locale.LocaleManager;
+
+public final class ConfluencePlugin implements ExecutorPlugin {
+
+	@Inject
+	public LocaleManager localeManager;
+	//String locale = paramManager.getParam("LOCALE").getValueAsString();
+
+	@Override
+	public void init() {
+		//TODO : initialize Locale
+		localeManager.add("io.vertigo.chatbot.engine.plugins.bt.confluence.multilingual.ConfluenceMultilingualResources", ConfluenceMultilingualResources.values());
+
+	}
```

Comment:
```suggestion
	@Inject
	public ConfluencePlugin(final LocaleManager localeManager, final ParamManager paramManager) {
		Assertion.check()
				.isNotNull(localeManager)
				.isNotNull(paramManager);
		//-----
		//String locale = paramManager.getParam("LOCALE").getValueAsString();
		//TODO : initialize Locale
		localeManager.add("io.vertigo.chatbot.engine.plugins.bt.confluence.multilingual.ConfluenceMultilingualResources", ConfluenceMultilingualResources.values());
	}
```
Et du coup supprimer le init des autres endroits également ;)